src/utils/service/service.js

```diff
@@ -71,6 +71,11 @@ class Service {
    * @return {Object} the response data
    */
   handleSuccess(response) {
+    if (response.data === '') {
```

Comment:
Could this be if `(!response.data) {`? Would then catch nulls and undefineds.

Reply:
Ah yes, had it this way when we spiked it in `responseTransform`